jobs/pipelines/powervm/zstreams/zstream-ocp4x-p8-min-next-next-upgrade/Jenkinsfile

```diff
@@ -0,0 +1,304 @@
+        PULL_SECRET = credentials('PULL_SECRET')
+        OS_AUTH_URL=credentials('URL_SCNLPOWERCLOUD')
+
+        //Parameters
+        OCP_RELEASE="${params.Release}"
+        UPGRADE_RELEASE="${params.UpgradeRelease}"
+        CONFIG_TYPE="min"
+        CURRENT_BUILD="${params.CurrentBuild}"
+        UPGRADE_BUILD="${params.UpgradeBuild}"
+        NEXT_UPGRADE_BUILD="${params.NextUpgradeBuild}"
+        TIMEOUT = "${params.KeepFor}"
+        ENABLE_E2E_TEST="true"
+        
+        //Env constants
+        HARDWARE_CHOSE = "P8"
+        AVAILABILITY_ZONE = "zstream"
+        TARGET = "deploy-openshift4-powervc"
+        TEMPLATE_FILE = ".${TARGET}.tfvars.template"
+        POWERVS = false
+        SCRIPT_DEPLOYMENT = false
+        WAIT_FOR_DEBUG = "0"
+        REDHAT_RELEASE = "8.3"
+        ENABLE_SCALE_TEST="false"
+    }
+
+    stages {
+        stage('Clone ansible extra') {
+            steps {
+                cloneRepo("git@github.ibm.com:redstack-power/ocp4_ansible_extras.git", "ansible_extra")
+            }
+        }
+        stage('Clone ansible ocp4-playbook') {
+            steps {
+              cloneRepo("https://github.com/ocp-power-automation/ocp4-playbooks.git", "ocp4_playbooks")
+            }
+        }
+        stage('Setup Common Environment Variables') {
+            steps {
+                setupCommonEnvironmentVariables()
+                setupClusterConfig("${CONFIG_TYPE}")
+                script {
+                    wrap([$class: 'BuildUser']) {
+                        env.INSTANCE_NAME = "rdr-zstnxt"
+                    }
+                    env.TERRAFORM_VER = "0.13.0"
+                    env.OS_TENANT_NAME="icp-test"
+                    env.OS_NETWORK = "workload"
+                    env.OS_PRIVATE_NETWORK = "workload"
+                    env.SCG_ID = "df21cec9-c244-4d3d-b927-df1518672e87"
+                    env.MOUNT_ETCD_RAMDISK = "false"
+                    env.VOLUME_STORAGE_TEMPLATE = "V7K base template"
+                    env.E2E_BRANCH="release-${env.UPGRADE_RELEASE}"
+                    env.UPGRADE_DELAY_TIME = "600"
+                    env.UPGRADE_PAUSE_TIME = "90"
+                    env.SETUP_SQUID_PROXY = "true"
+                    env.E2E_EXCLUDE_LIST="https://raw.github.ibm.com/redstack-power/e2e-exclude-list/${env.UPGRADE_RELEASE}-powervm/ocp${env.UPGRADE_RELEASE}_power_exclude_list.txt"
+                    if ( env.OCP_RELEASE == "4.6"||env.OCP_RELEASE == "4.7")  {
+                        env.OPENSHIFT_POWERVC_GIT_TF_DEPLOY_BRANCH="release-${env.OCP_RELEASE}" //The download branch
+                    }
+                    else {
+                        env.OPENSHIFT_POWERVC_GIT_TF_DEPLOY_BRANCH="master" //The download branch
+                    }
+                    if ( env.UPGRADE_RELEASE == "4.6"||env.UPGRADE_RELEASE == "4.7")  {
+                        env.GOLANG_TARBALL = "https://dl.google.com/go/go1.15.2.linux-ppc64le.tar.gz"
```

Comment:
@varad-ahirwadkar This should be next upgrade release since you are running E2e after next upgrade

Reply:
ok sure